### DISCUSSION TITLE
Chromium Embedded Framework

### DISCUSSION THREAD

**Discussion by maksw2** [Sectonidse/SourceDEFUN#20] [Feature Requests]
For UI. What do you think?

**Comment by Sectonid** [comment]
Great idea! I'm good at using CSS styles, so i'll make a theme for SourceDEFUN.

**Reply by maksw2** [reply]
Lambda Wars has a good implementation of that. Also, it has better diffuse lighting.

**Reply by Sectonid** [reply]
> Also, it has better diffuse lighting.

I checked their source code, looks like they have deferred lighting implementation too!

**Reply by maksw2** [reply]
![image](https://github.com/user-attachments/assets/6d357aae-f023-4e0b-aee1-d8b5da34ead9)
tried to load a SwarmDeferred map and it kinda works. I'm talking about their implementation because it preserves lightmaps.

**Reply by maksw2** [reply]
![image](https://github.com/user-attachments/assets/aa3aecb0-7d50-44b6-8702-8aa7fed87e44)
it's not stable
